cassovary-collections/README.md

```diff
@@ -0,0 +1,9 @@
+# Cassovary-collections
+
+This subproject contains specialized collections that are used by Cassovary library.
+
+General goals for the collections implemented in this subproject are:
+
+1) low memory footprint (storing primitive types where possible); 
+2) optimization for Int and Long types (no boxing);
```

Comment:
may be refer to a benchmark file specifically comparing CSeq to IndexedSeq for ints / longs ?